kubecost-cloud-gcp-marketplace-license.md

```diff
@@ -0,0 +1,36 @@
+# Kubecost Cloud GCP Marketplace Licensing
+
+Kubecost Cloud is [available for licensing on GCP Marketplace](https://console.cloud.google.com/marketplace/product/kubecost-public/kubecost-cloud) and can be installed in minutes. This guide will take you through licensing through GCP Marketplace, and next steps for setting up Kubecost Cloud. Kubecost currently offers 30 days of Kubecost Cloud free without licensing fees.
+
+{% hint style="info" %}
+Licensing Kubecost Cloud through GCP Marketplace will not directly integrate your GCP billing data into your Kubecost Cloud environment. For more information, see our [GCP Cloud Integration guide](/kubecost-cloud/kubecost-cloud-cloud-billing-integrations/kubecost-cloud-gcp-integration.md).
+{% endhint %}
+
+## Prerequisites
+
+* Set up a Google Cloud account with an attached [billing account](https://cloud.google.com/billing/docs/how-to/create-billing-account).
+* [Set up a Kubecost Cloud account](kubecost-cloud/cloud-installation-and-onboarding.md#creating-a-user-account)
+
+## GCP Marketplace licensing guide
+
+On the [Product details page for Kubecost](https://console.cloud.google.com/marketplace/product/kubecost-public/kubecost-cloud), select _Subscribe_. You will be taken to an Order Summary page.
+
+![Order Summary page](/images/order-summary.png)
+
+Under "1. Select Plan", the default plan should be Cloud Pro, and the default usage fee should be USD 0.167 per node per day. You can use the Pricing Calculator in the right sidebar to determine estimated costs by providing estimated timeframe of usage from 1 day to 1 year, and total node count.
```

Comment:
We don't typically call out pricing in docs. I suggest we remove it here.

Reply:
Although it is in the images as well.

Reply:
Yes I mention it because it is part of the Pricing Calculator included in the licensing procedure.